### PR TITLE
virsh_cpu_compare: Add a new cpu element if it not exist

### PR DESCRIPTION
**Pull request by will-Do**
Signed-off-by: Yanbing Du <ydu@redhat.com>

### CODE REVIEW

**Comment by PandaWei** [comment]
Look good, ACK

File: libvirt/tests/src/virsh_cmd/domain/virsh_cpu_compare.py [hunk 1]
```diff
@@ -89,6 +89,12 @@ def get_cpu_xml(target, mode):
     try:
         # Prepare VM cpu feature if necessary
         if modify_target in ['feature_name', 'feature_policy', 'delete']:
+            try:
```
Comment (chuanchang) [inline libvirt/tests/src/virsh_cmd/domain/virsh_cpu_compare.py]: The exception process may be not necessary in here, it should be enough to use 'if not vmxml.get('cpu'):' as a condition.

Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/domain/virsh_cpu_compare.py]: Oh, yes, I will update it, thanks.

**Comment by waynesun09** [comment]
ACK and merging now, thx!

**Pull request merged**